contracts/non-fungible-tokens/andromeda-auction/src/contract.rs

```diff
@@ -310,11 +379,26 @@ fn execute_update_auction(
         ContractError::StartTimeAfterEndTime {}
     );
 
+    let uses_whitelist = if let Some(ref whitelist) = whitelist {
+        for whitelisted_address in whitelist {
```

Comment:
USE `permission_action` on the auction id and remove the `uses_whitelist` check in bid, replace `is_permissioned_strict` with `is_permissioned`.